cli/src/main/java/org/opentosca/toscana/cli/ApiController.java

```diff
@@ -364,30 +233,21 @@ public String listTransformation(String csar) throws IOException {
      @param csar CSAR, for which Transformation Info should be shown
      @param plat Platform, for which Information should be shown
      @return output for the CLI
-     @throws IOException if the responsebody is null
      */
-    public String infoTransformation(String csar, String plat) throws IOException {
-        Call<Transformation> transformationCall = service.getTransformation(csar, plat);
-        Response<Transformation> response = transformationCall.execute();
-
-        if (response.code() == 200) {
-            Transformation transformation = response.body();
-            if (transformation.getPlatform() != null) {
-                return con.TRANSFORMATION_INFO_SUCCESS + csar + " on Platform: "
-                    + plat + "\nPlatform: " + transformation.getPlatform()
-                    + "\nProgress: " + transformation.getProgress() + "\nStatus: " + transformation.getStatus();
-            } else {
-                return con.TRANSFORMATION_INFO_EMPTY;
-            }
-        } else if (response.code() == 404) {
-            if (response.errorBody().string() != null) {
-                return con.TRANSFORMATION_INFO_ERROR404 + "\n" + response.errorBody().string();
-            } else {
-                return con.TRANSFORMATION_INFO_ERROR404;
-            }
-        } else {
-            return con.TRANSFORMATION_INFO_ERROR;
+    public String infoTransformation(String csar, String plat) {
+        Transformation transformation;
+        StringBuilder stringTransformation = new StringBuilder();
+        try {
+            transformation = toscAnaAPI.getTransformation(csar, plat);
+
+            stringTransformation.append(transformation.getPlatform()).append(", ").append(transformation.getProgress()).append(", ").append(transformation.getStatus());
```

Comment:
Please avoid such long lines of code by adding a newline between the appends. Maybe put every `.append(transformation.<something>()).append(", ")` in one seperate line